### PR TITLE
add stricter linting to match 'symbol-sdk-core-python'

### PR DESCRIPTION
**Pull request by Jaguar0625**

### CODE REVIEW

File: scripts/generate_all.sh [hunk 1]
```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
-source "$(dirname $0)/schema_lists.sh"
-source "$(dirname $0)/generate_batch.sh"
+# shellcheck source=.
+# shellcheck disable=SC1091
+source "$(dirname "$0")/schema_lists.sh"
+source "$(dirname "$0")/generate_batch.sh"
```
Comment (Jaguar0625) [inline scripts/generate_all.sh:6]: you might want to doublecheck this

File: lint.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+find . -type f -name "*.sh" -print0 | xargs -0 shellcheck
+find . -type f -name "*.py" -print0 | PYTHONPATH=. xargs -0 "$(pyenv which isort)" --check-only --line-length 140
+find . -type f -name "*.py" -print0 | PYTHONPATH=. xargs -0 "$(pyenv which pycodestyle)" --config=.pycodestyle
+find . -type f -name "*.py" -print0 | PYTHONPATH=. xargs -0 "$(pyenv which pylint)" --load-plugins pylint_quotes
```
Comment (gimer) [inline lint.sh:6]: @Jaguar0625 why `pyenv which foo` instead of `which foo` ?

File: scripts/generate_all.sh (hunk 1 above)
Comment (gimer) [reply scripts/generate_all.sh:6]: yes, it's actually best way iirc

**Review by gimer — APPROVED** [review]

File: lint.sh (hunk 2 above)
Comment (Jaguar0625) [reply lint.sh:6]: `pyenv which foo` seems to return correct (different) binary

**Pull request merged**